starcounter-include.html

```diff
@@ -476,13 +519,25 @@
          * @param {DocumentFragment} givenComposition
          */
         StarcounterIncludePrototype.stampComposition = function(givenComposition){
```

Comment:
I would make this `async function(givenComposition) {`. Justification below.

Reply:
below, meaning where?

Reply:
The comment was not posted, but we discussed it in person.

Reply:
I gave it a second thought. The function name and main responsibility is to "stampComposition" this stamping is always fully synchronous, that's why "stamped" (I'd rename it to "presentation-stamped") is called synchronously at the end.

"load composition" is the action that **may** be asynchronous. I'd trigger an `presentation-loaded` after it's done. WDYT?

Reply:
I think the event will be fine.